huggingface/base/inference/artifacts/config.properties

```diff
@@ -0,0 +1,5 @@
+vmargs=-XX:+UseContainerSupport -XX:InitialRAMPercentage=8.0 -XX:MaxRAMPercentage=10.0 -XX:-UseLargePages -XX:+UseG1GC -XX:+ExitOnOutOfMemoryError
```

Comment:
I would this file place in `build_artifacts` instead of `artifacts` to keep it the same across all frameworks.
https://github.com/vdantu/deep-learning-containers/tree/hugging_face/pytorch/inference/docker/build_artifacts

Also, can you remove the `__init__.py` from `artifacts`